### PR TITLE
Feature Card component

### PR DESCRIPTION
**Pull request by nashirjamali**
image
Create feature component

### CODE REVIEW

**Comment by baralogi** [comment]
image

i think the margin is too large

**Comment by nashirjamali** [comment]
> image
> 
> i think the margin is too large

test

File: pages/index.jsx [hunk 1]
```diff
@@ -1,14 +1,54 @@
...
 import Body from '../layouts/Body';
 
 function Index() {
   return (
     <Body title="Home">
-      <Box>
-        <Heading>Ini Halaman Home</Heading>
-        <Text>Hello</Text>
-      </Box>
+      <Stack direction={['column', 'row']} spacing="24px" align="stretch">
```
Comment (nashirjamali) [inline pages/index.jsx:10]: @ssembara updated bro,, I changed it to stack component

**Review by baralogi — APPROVED** [review]
nice 

closed #4

**Pull request merged**